lib/orchestrators/join.js

```diff
@@ -17,10 +17,11 @@ const join = (dispatch) => {
 
       // TODO better checking if task/join/junction/fork
       if (task.info) {
-        console.log('Joining to task: ' + task.info.name)
+        if (task.info.props) {
```

Comment:
So, if I do `console.log('Joining to task: ' + _.get(task, 'info.props.name'))` it seems to be similar to what it was previously, but without the problem of `task.info` being undefined and breaking the run before it ended, right?